mitosheet/src/mito/components/toolbar/Toolbar.tsx

```diff
@@ -79,6 +82,9 @@ export const Toolbar = (
             })}
         </div>
     }
+    if (props.uiState.currOpenTaskpane.type === TaskpaneType.GRAPH) {
```

Comment:
Another thing you could here is check that the open graph has the same graph id as the params from the steps list in the analysis data. So that we make sure we're accessing the right params.

Reply:
Wouldn't we want to add that in the `ChartDesignTabContents`? As long as we have a graph tab open we'll want this tab to exist.

Reply:
I think it's the same, but it's also unnecessary I think.